Main_Solution/main_solution_test.rb

```diff
@@ -135,10 +146,13 @@ def test_conditional_watch_issue
     end
 
     @wait.until{is_issue_watched?}
-    assert(is_issue_watched?)
+    expect(is_issue_watched?).to be
```

Comment:
Yep, I'm expecting to see what I was waiting for, despite if it won't be seen timeout will be reached.
Not a nice solution definitely =)
Removed excessive expectation, it's enough to check user appeared in 'watchers' list to see 'watching' feature works (which is done at the bottom).
Fixed.